compiler-plugin/pom.xml

```diff
@@ -22,14 +22,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <groupId>org.wso2.ei.b7a</groupId>
+        <groupId>org.ballerinalang</groupId>
```

Comment:
Shouldn't we change the package name too? https://github.com/ballerina-platform/module-ballerina-java.jms/tree/master/utils/src/main/java

Reply:
Fixed in 7baaef10365905864a27b633efad933762ad0b31